charts/nginx-ingress/templates/_helpers.tpl

```diff
@@ -312,6 +312,14 @@ List of volumes for controller.
 - name: nginx-log
   emptyDir: {}
 {{- end }}
+{{- if .Values.controller.appprotect.v5 }}
+- name: app-protect-bd-config
+  emptyDir: {}
+- name: app-protect-config
+  emptyDir: {}
+- name: app-protect-bundles
+  emptyDir: {}
+{{- end }}
```

Comment:
Needs to support BYO-PVC (we probably don't want to own the PVC, just reference an existing one)